src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/Toolbar.js

```diff
@@ -11,7 +14,41 @@ export default class Toolbar extends React.PureComponent {
         return (
             <header className={toolbarStyles.toolbar}>
                 <nav>
-                    {toolbarStore.items.map((item) => (<Item key={item.title} {...item} />))}
+                    <div className={toolbarStyles.pullLeft}>
+                        {toolbarStore.hasBackButtonConfig() &&
+                            <BackButton {...toolbarStore.getBackButtonConfig()} />
+                        }
+                        <div className={toolbarStyles.buttons}>
+                            {toolbarStore.hasButtonsConfig() &&
+                                toolbarStore.getButtonsConfig().map((buttonConfig) => {
+                                    if (buttonConfig.options) {
+                                        return <DropdownButton key={buttonConfig.value} {...buttonConfig} />;
+                                    } else {
+                                        return <DefaultButton key={buttonConfig.value} {...buttonConfig} />;
+                                    }
+                                })
+                            }
+                        </div>
+                    </div>
+                    <div className={toolbarStyles.pullRight}>
+                        {toolbarStore.hasIconsConfig() &&
+                            <div className={toolbarStyles.icons}>
+                                {
+                                    toolbarStore.getIconsConfig().map((icon) => (
+                                        <Icon key={icon} name={icon} className={toolbarStyles.icon} />
```

Comment:
Is that enough? Don't we add a bit more e.g. to display the green dot in the icon currently placed there? Or do these icons already contain stuff like that? (Can't really imagine, since it's a plain icon font, with the same color all over the place) Or did Tim say something that the color will be abandoned here?